### DISCUSSION TITLE
how to skip folder file backing and restoring

### DISCUSSION THREAD

**Discussion by vfolk30** [k8up-io/k8up#808] [Q&A]
Hi ,
There is any possibility to  exclude the folder/directory when you are restoring ?

I am getting below issue because my  .snapshot directory is  read-only

ignoring error for /data/myclaim/.snapshot: UtimesNano: read-only file system

There is any way  in restore job to  provide arguement to exclude the directory

**Comment by Kidswiss** [comment]
Hi @vfolk30 

We currently have an issue open for excluding files during backups: https://github.com/k8up-io/k8up/issues/317

You mention that you want to exclude it during restore. Does that mean you want it included in the backup anyway? Because if it's not backed up in the first place you won't have to exclude it during the restore. :)

**Reply by vfolk30** [reply]
you only see the problem when you do restoration.
I am happy if I can exclude during the backup then it wont make any problem for restore.

Any  progress on  (https://github.com/k8up-io/k8up/issues/317)

**Reply by Kidswiss** [reply]
We haven't started with #317 yet, but your input helps us to prioritize the features better.

**Reply by vfolk30** [reply]
In my case I can not  disable .snapshot directory from my nfs  storage.
Such features can help users and performance would be  better .

**Reply by vfolk30** [reply]
Could you also check the document:
https://k8up.io/k8up/2.5/how-tos/restore.html

it seems in restore example s3> pvc has different   claimname in spec.
